cmd/launcher/svc_windows.go

```diff
@@ -190,3 +194,43 @@ func (w *winSvc) Execute(args []string, r <-chan svc.ChangeRequest, changes chan
 		}
 	}
 }
+
+const (
+	launcherServiceRegistryKeyNameFormat = `SYSTEM\CurrentControlSet\Services\Launcher%sSvc`
+
+	// DelayedAutostart is type REG_DWORD, i.e. uint32. We want to turn off delayed autostart.
+	delayedAutostartName         = `DelayedAutostart`
+	delayedAutostartValue uint32 = 0
+
+	// DependOnService is type REG_MULTI_SZ, i.e. a list of strings
+	dependOnServiceName = `DependOnService`
+	dependOnService     = `Dnscache`
+)
+
+func editRegistry(logger log.Logger) {
+	// Get launcher service key
+	// TODO: ideally we'd pull the identifier from somewhere legitimate
+	launcherServiceKeyName := fmt.Sprintf(launcherServiceRegistryKeyNameFormat, "KolideK2")
```

Comment:
Can we pull it from the config file?

Reply:
My thought was that we'd extract it from the server hostname in the options, but hadn't spent the time chasing that down yet to make sure that was correct. If yes I can definitely do that, the options are available here.